backend/pkg/config/config_test.go

```diff
@@ -0,0 +1,96 @@
+package config_test
```

Comment:
Awesome! Thanks for the tests.
I wonder if we could cover all flags before and after, just to make sure we don't have to test all of them manually.

Reply:
We will have to set up integration tests for the before and after, also we won't be able to test all the config values in integration test apart from things like PORT, etc. Unfortunately, I think we have to settle with manual tests for this once.